content/es/docs/tasks/debug-application-cluster/debug-init-containers.md

```diff
@@ -104,25 +100,21 @@ Los Init Containers que ejecutan secuencias de línea de comandos muestran los c
 conforme se van ejecutando. Por ejemplo, puedes hacer lo siguiente en Bash
 indicando `set -x` al principio de la secuencia.
 
-{{% /capture %}}
-
-{{% capture discussion %}}
+<!-- discussion -->
 
 ## Comprender el estado de un Pod
 
 Un estado de un Pod que comienza con `Init:` especifica el estado de la ejecución de
 un Init Container. La tabla a continuación muestra algunos valores de estado de ejemplo
 que puedes encontrar al depurar Init Containers.
 
-Estado | Significado
------- | -------
-`Init:N/M` | El Pod tiene `M` Init Containers, y por el momento se han completado `N`.
-`Init:Error` | Ha fallado la ejecución de un Init Container.
-`Init:CrashLoopBackOff` | Un Init Container ha fallado de forma repetida.
-`Pending` | El Pod todavía no ha comenzado a ejecutar sus Init Containers.
-`PodInitializing` o `Running` | El Pod ya ha terminado de ejecutar sus Init Containers.
-
-{{% /capture %}}
+| Estado                        | Significado                                                               |
+| ----------------------------- | ------------------------------------------------------------------------- |
+| `Init:N/M`                    | El Pod tiene `M` Init Containers, y por el momento se han completado `N`. |
+| `Init:Error`                  | Ha fallado la ejecución de un Init Container.                             |
+| `Init:CrashLoopBackOff`       | Un Init Container ha fallado de forma repetida.                           |
+| `Pending`                     | El Pod todavía no ha comenzado a ejecutar sus Init Containers.            |
+| `PodInitializing` o `Running` | El Pod ya ha terminado de ejecutar sus Init Containers.                   |
```

Comment:
:warning: These changes are not related to capture shortcodes.

Reply:
Markdown formatting on save, but I will close this PR and approve yours https://github.com/kubernetes/website/pull/22492.